src/screens/Wallet.js

```diff
@@ -121,10 +249,18 @@ class Wallet extends React.Component {
       <div>
         {!this.state.backupDone && renderBackupAlert()}
         <div className="content-wrapper">
+         {/* This back button is not 100% perfect because when the user has just unlocked the wallet, it would go back to it when clicked
+           * There is no easy way to get the previous path
+           * I could use a lib (https://github.com/hinok/react-router-last-location)
+           * Or handle it in our code, saving the last accessed screen
+           * XXX Is it worth it to do anything about it just to prevent this case?
+           */}
+          <BackButton {...this.props} />
```

Comment:
I didn't understand the problem. Can you give me an example, please?

Reply:
1. Wallet is locked (user is in the locked screen)
2. User unlocks the wallet and goes to the main screen (token history)
3. User clicks 'Back' and goes to locked screen again.

To prevent this case we have to install a lib, so we can have access to the last path.

Reply:
I see. Maybe you can include this example in the comment to make it easier for future readers.

About the problem itself, can't the lock screen just check that the wallet is unlocked and "go back again", instead of adding a new dependency?

Reply:
I can try it, but feel like the page will 'blink' and I really don't like it. But maybe not. Will try it.

Reply:
Make sense. Blinking is an undesirable side effect. Your solution seems better.